q2cli/core/usage.py

```diff
@@ -0,0 +1,73 @@
+import qiime2.sdk.usage as usage
+
+
+class CLIUsage(usage.Usage):
+    def __init__(self):
+        super().__init__()
+        self._imports = set()
+        self._recorder = []
+        self._init_data_refs = dict()
+
+    def _init_data_(self, ref, factory):
+        self._init_data_refs[ref] = factory
+        # Don't need to compute anything, so just pass along the ref
+        return ref
+
+    def _merge_metadata_(self, ref, records):
+        first_md = records[0].ref
+        remaining_records = ', '.join([r.ref for r in records[1:]])
+        t = '%s = %s.merge(%s)\n' % (ref, first_md, remaining_records)
+        self._recorder.append(t)
+        return ref
+
+    def _get_metadata_column_(self, ref, record, column_name):
+        t = '%s = %s.get_column(%r)\n' % (ref, record.ref, column_name)
+        self._recorder.append(t)
+        return ref
+
+    def _comment_(self, text: str):
+        self._recorder.append('# %s' % (text,))
+
+    def _action_(self, action: usage.UsageAction, input_opts: dict, output_opts: dict):
+        action_f, action_sig = action.get_action()
+        # TODO: Probably don't need this
```

Comment:
Correct, no need to track imports, since that concept doesn't apply to this interface.